packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-listener.ts

```diff
@@ -363,6 +363,7 @@ export class ApplicationListener extends BaseListener implements IApplicationLis
       healthCheck: props.healthCheck,
       port: props.port,
       protocol: props.protocol,
+      protocolVersion: props.protocolVersion,
```

Comment:
What do you think about adding `protocolVersion` to `AddApplicationTargetsProps` as well?

Reply:
Are you referring to this?

https://github.com/aws/aws-cdk/pull/14092/files/eff234b0407be3d2f1cb1e8b501b0fa48bf0d056#diff-87cb97a4a8916da7963bded49279e74120b58fd83b956028106159f6518fc761R798-R812

Reply:
Oh yeah, sorry, must have mis-read the diff as-is. This looks great then.